app/Website/routes/website-routes.php

```diff
@@ -15,3 +16,14 @@
 Route::get('/blog-post-tags/{tag}', [BlogPostTagsController::class, 'show'])->name('blog-post-tags.show');
 
 Route::post('/contact', ContactController::class)->name('contact')->middleware('throttle:5,1');
+
+Route::post('newsletter/store', [NewsletterSubscribeController::class, 'store'])->middleware('throttle:5,1');
```

Comment:
`/newsletter-subscribers` is a better route URI. Also rename the controller from `NewsletterSubscribeController` to `NewsletterSubscriberController`

Reply:
Changed